### DISCUSSION TITLE
Import a markdown file and render

### DISCUSSION THREAD

**Discussion by akshaybabloo** [vuejs/vitepress#2952] [Q&A] [answered]
Is there a way to import a markdown file and render it as HTML? The reason for this is, I have a bash script that generates a new file every time it's run.

Importing something similar to https://vitepress.dev/guide/markdown#import-code-snippets

**Comment by brc-dd** [comment, marked as answer]
https://vitepress.dev/guide/markdown#markdown-file-inclusion
